dist/codecov.sh

```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/env bash
```

Comment:
you can do a shebang for `sh` - would recommend as otherwise I believe it just uses the default shell, which could potentially be bad? Regardless would rather be explicit about the API necessary for the script to run